appengine/flexible/StaticContent/README.md

```diff
@@ -3,53 +3,7 @@
 This sample application demonstrates how to serve static content with
 Google App Engine Flexible Environment.
 
-## Prerequisites
+For detailed steps to deploy this app, check out
+[the Google Cloud
+documentation.](https://cloud.google.com/appengine/docs/flexible/serving-static-files?tab=.net)
```

Comment:
nit: [the Google Cloud documentation.](https://...) -> [the Google Cloud
documentation](https://...).